index.rst

```diff
@@ -44,14 +44,579 @@
+It is anticipated that artifacts generated from automated on-the-fly analyses will be stored in this area.
+An example of this would be the `Papermill Executed Parameterized Notebooks`_.
+
+
+Chronograf
+''''''''''
+
+The `summit-based Chronograf interface <https://chronograf-summit-efd.lsst.codes/>`_ (VPN required) provides a user-friendly graphical interface to the each `deployed instance of the EFD <https://sqr-034.lsst.io/#efd-deployments>`_.
+It is particularily useful for creating visualization dashboards to show the current status of the observatory when the LOVE functionality is either not yet functional or simply not planned to be implemented.
+It is not well suited for complex queries or figures and previous queries/plots are not easily replicated.
+Furthermore, it always displays the last event seen.
+Therefore if a CSC crashes, it will always show the last published state.
+For this reason (and many others), it's not an appropriate substitute for a true status GUI, such as what is being provided by the `LOVE`_ interface.
+
+Watcher
+'''''''
+
+.. Patrick writing this
+
+The `Watcher CSC <https://ts-watcher.lsst.io/>`_ monitors control components listening for data that signals an alarm to the observer.
+The alarms are defined by a series of "rules" which are defined and added to the package.
+The CSC itself is not a display tool nor does it have any display functionality.
+When condition defined by an rule is met, an alarm is generated and the observer is alerted via a LOVE screen.
+The alarm has a series of levels and audible alerts are sent out via LOVE, as well as a visual notification.
+Once the alarm is acknowledged by the observer then alert is considered to be completed.
+There is no feedback or interaction for the observer beyond the acknowledgment to the alarm.
+
+SAL Scripts
+'''''''''''
+
+.. Patrick writing this
+
+`SAL scripts <https://ts-salobj.lsst.io/sal_scripts.html#lsst-ts-salobj-sal-scripts>`_ are a series of coordinated sequences, often consisting of commands to CSCs, that are executed by the `ScriptQueue <https://ts-scriptqueue.lsst.io/>`_.
+It is anticipated that most standard operations will utilize scripts.
+Also possible, although not standard practice, is to manually execute a script from the `Nublado (Jupyter Notebook) Interface`_.
+From within a SAL script, users can send standard commands to components as well as send data to the `OCPS`_ for processing.
+The script can then either wait for the analysis to complete and continue, with the ability to act based on the result, or launch the process (e.g. image reduction) and continue executing the script.
+Scripts are not intended to perform any data analysis and do not produce artifacts.
+They can not display any figures nor report customized results (only status).
+The monitoring and execution of a SAL Scripts progress is done via the LOVE ScriptQueue GUI.
```

Comment:
"The monitoring and execution of a SAL Scripts progress is" Check grammar and intended meaning here. It seems that the intended meaning is that LOVE can be used to initiate the execution of a SAL script, and LOVE allows monitoring the progress of execution.

Reply:
Ya, that's confusing. I removed it and added a line 249 (The LOVE system contains a graphical ScriptQueue interface to control and monitor the execution of scripts.)